README.md

```diff
@@ -181,6 +182,18 @@ client.checkout.fetch(checkoutId).then((checkout) => {
 });
 ```
 
+### Adding a Discount
+```javascript
+const checkoutId = 'Z2lkOi8vc2hvcGlmeS9Qcm9kdWN0SW1hZ2UvMTgyMTc3ODc1OTI='; // ID of an existing checkout
+const discountCode = 'best-discount-ever';
+
+// Add an item to the checkout
```

Comment:
this isn't adding an item to the checkout. This is adding a discount to the checkout